bcipy/helpers/convert.py

```diff
@@ -2,18 +2,22 @@
 
 import logging
 import os
+import tarfile
 from pathlib import Path
 from typing import Dict, List, Tuple
 
 import numpy as np
 from pyedflib import FILETYPE_EDFPLUS, EdfWriter
+from tqdm import tqdm
 
 from bcipy.helpers.load import load_json_parameters, load_raw_data, extract_mode
 from bcipy.helpers.triggers import trigger_decoder, apply_trigger_offset, trigger_durations
 
 
 logger = logging.getLogger(__name__)
 
+FILE_LENGTH_LIMIT = 150
```

Comment:
This could be an input argument with a default, but this seems okay too since it's a warning